src/main/java/hudson/plugins/analysis/core/BuildHistory.java

```diff
@@ -348,5 +351,33 @@ public BuildHistory(final AbstractBuild<?, ?> baseline, final Class<? extends Re
     public BuildHistory(final AbstractBuild<?, ?> baseline, final Class<? extends ResultAction<? extends BuildResult>> type) {
         this(baseline, type, false);
     }
+
+    /**
+     * Creates a new instance of {@link BuildHistory}.
+     *
+     * @param baseline
+     *            the build to start the history from
+     * @param type
+     *            type of the action that contains the build results
+     * @param usePreviousBuildAsReference
+     *            determines whether the previous build should always be used
+     *            as the reference build
+     * @param useStableBuildAsReference
+     *            determines whether only stable builds should be used as
+     *            reference builds or not
+     * @since 1.66
+     * 
+     * @deprecated use {@link #BuildHistory(Run, Class, boolean, boolean)}
+     */
+    @Deprecated
+    public BuildHistory(final AbstractBuild<?, ?> baseline,
+            final Class<? extends ResultAction<? extends BuildResult>> type,
+            final boolean usePreviousBuildAsReference,
+            final boolean useStableBuildAsReference) {
```

Comment:
BTW it is easier for reviewers to see that you are maintaining binary compatibility when the old deprecated member is adjacent to the new one, forming one diff hunk.